.env-example

```diff
@@ -15,4 +15,5 @@ OPENAI_API_KEY  = 1234567890
 
 # Bot constants
 TEAM_NAME = 'Boca Jrs'
+TEAM_COLORS = 'azul y amarrillo'
```

Comment:
Change to TEAM_COLORS = 'Azul y Amarillo'

Reply:
Done!